README.md

```diff
@@ -35,33 +35,28 @@ Once the environment is created, activate it:
 conda activate pytom_tm
 ```
 
-Then clone the repository and install it with pip (building cupy can take a while!): 
+Then install the code with `pip` (building cupy can take a while!):
 
 ```commandline
-git clone https://github.com/SBC-Utrecht/pytom-match-pick.git
-cd pytom-match-pick
-python -m pip install '.[plotting]'
+python -m pip install pytom-match-pick[plotting]
 ```
 
 The installation above also adds the optional dependencies `[matplotlib, seaborn]` which are required to run 
 `pytom_estimate_roc.py`. They are not essential to the core template matching functionality, so for some systems 
 (such as certain cluster environments) it might be desirable to skip them. In that case remove `[plotting]` from the pip install command:
 
 ```commandline
-python -m pip install .
+python -m pip install pytom-match-pick
 ```
 
-## Tests
-
-To run the unittests:
-
+## Cupy warning
+Trying to import the code with 
 ```commandline
-cd tests
-python -m unittest discover
+python -c "import pytom_tm"
 ```
 
-If cupy is not correctly installed, this should raise some errors:
-
+can show a cupy warning. If this is the case, this probably means cupy is not correctly installed.
```

Comment:
```suggestion
will show an import error. If this is the case, this probably means cupy is not correctly installed.

```

Reply:
Currently the code only does logging.warning, not an actually import error

Reply:
Fair!